publish.css

```diff
@@ -1,23 +1,112 @@
 /* This Theme is called "Reverie" and Obsidian Theme created by Santi Younger */
-
-Last Update:
-2021-03-11 12:06
-
 @import url('https://fonts.googleapis.com/css2?family=Modern+Antiqua&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Raleway&display=swap');
 
-:root {
-  --font-monospace: 'Source Code Pro', monospace;
-
+body {
+  --font-monospace: 'Source Code Pro';
   --wws-font-modern: "Modern Antiqua";
-
   --wws-font-raleway: "Raleway";
-}
-   
+  --wws-font-text: 'Modern Antiqua';
+  --wws-font-monospace: 'Source Code Pro', Jetbrains Mono;
+  --wws-header-font: 'Modern Antiqua';
+  --wws-header-font-caps-variant: normal;
+  --wws-font-tags: var(--wws-header-font);
+  --wws-font-callout-title: var(--wws-header-font);
+
+  --h1-font: var(--wws-header-font);
```

Comment:
We'll use h1-h6 variables found further down in app.css to force the headers to be the font we want.